source/helpers/test/CaseDataConverter.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { replaceTagPart } from '../CaseDataConverter';
+
+describe('replaceTagPart', () => {
+  
```

Comment:
Also notice that you are declaring the input variable in every tests, which makes it a little DRY.
This can be refactored to something like this:

```js
// The beforeEach function will be run before every it-test is run. 

let input;

beforeEach(() => {
  input = {
  tag: 'test:example:replace',
  part: 'replace',
  value: 'success'
  };
});

// And if you want to change a property in one test, then you only need to change that property value instead of re-creating the entire input constant like:
describe('some description', () => {
  it('one awesome test', () => {
    input.tag = 'my new value';
  });
});
```